### PR TITLE
Use LangVersion=8 where feasible in testing 

### PR DESCRIPTION
**Pull request by jskeet**
Note that now both MaybeNull1Attribute and MaybeNull2Attribute fail to build - I believe this is correct, and should be reflected in spec changes which you may be doing already. Happy to wait until you're done, then rebase this.

### CODE REVIEW

**Comment by BillWagner** [comment]
trying a close and reopen. #1218 has already been merged, and that removes the attributes added in 9.

Since there aren't any conflicts, that may be enough.

**Review by BillWagner — APPROVED** [review]
This LGTM @jskeet 

Once the build is clean, let's :shipit:

**Comment by jskeet** [comment]
I'll rebase and repush.

File: standard/attributes.md [hunk 1]
```diff
@@ -1009,7 +1012,7 @@ Specifies that a non-nullable return value may be null.
 
 > *Example*: Consider the following generic method:
 >
-> <!-- Example: {template:"code-in-class-lib-without-using", name:"MaybeNull1Attribute", replaceEllipsis:true, customEllipsisReplacements: ["return default;"]} -->
+> <!-- Example: {template:"code-in-class-lib", name:"MaybeNull1Attribute", replaceEllipsis:true, customEllipsisReplacements: ["return default;"]} -->
```
Comment (BillWagner) [inline standard/attributes.md]: I think that in this sample, we should just expect `CS8627`. 

```suggestion
> <!-- Example: {template:"code-in-class-lib", name:"MaybeNull1Attribute", replaceEllipsis:true, customEllipsisReplacements: ["return default;"]", expectedErrors:["CS8627"]} -->
```

**Review by BillWagner — APPROVED** [review]
I have a suggestion to fix the build issue. Then, I think this is ready.

**Comment by jskeet** [comment]
Merged the suggestion - let's see whether we get to green...

File: standard/attributes.md [hunk 2]
```diff
@@ -1009,7 +1012,7 @@ Specifies that a non-nullable return value may be null.
 
 > *Example*: Consider the following generic method:
 >
-> <!-- Example: {template:"code-in-class-lib-without-using", name:"MaybeNull1Attribute", replaceEllipsis:true, customEllipsisReplacements: ["return default;"]} -->
+> <!-- Example: {template:"code-in-class-lib", name:"MaybeNull1Attribute", replaceEllipsis:true, customEllipsisReplacements: ["return default;"]", expectedErrors:["CS8627"]} -->
```
Comment (jskeet) [inline standard/attributes.md]:
```suggestion
> <!-- Example: {template:"code-in-class-lib", name:"MaybeNull1Attribute", replaceEllipsis:true, customEllipsisReplacements: ["return default;"], expectedErrors:["CS8627"]} -->
```

**Comment by jskeet** [comment]
Fixed a typo, so let's wait another time...

**Comment by jskeet** [comment]
It's green :) Merging.

**Pull request merged**